seed-job/template.yaml

```diff
@@ -54,7 +54,11 @@ objects:
                     def image_name = "${APP_ID}/${JOB_ID}:${DESIRED_TAG}"
                     def image_name_with_registry = "${REGISTRY_URL}/${image_name}"
                     def daemonset_name = "scan-data_scan-data"
+                    // setting this to ensure the user is notificatied only after the mandatory stages are complete
+                    // i.e. (checkout, prebuild) - lint - build - scan - deliver
                     def success = false
+                    // setting this to ensure the image is removed from the system if its locally built
```

Comment:
`s/its/it's`

Reply:
will take care of this docstring change as part of parameters ordering, as this fix will re-trigger CI, which might need multiple triggers to go green and we have resolution on other comment.